app/build.gradle.kts

```diff
@@ -111,6 +131,7 @@ dependencies {
     implementation("io.github.fornewid:material-motion-compose-core:0.10.3")
     implementation("com.google.dagger:hilt-android:$hiltVersion")
     annotationProcessor("com.google.dagger:hilt-android-compiler:$hiltVersion")
+    annotationProcessor ("com.google.dagger:hilt-compiler:$hiltVersion")
```

Comment:
Why need to apply `hilt-compiler` here?

Reply:
The apk didn't launch , encountered classnot found exception , I tried many way nothing worked.

It was working after adding kapt compiler.
kapt ("com.google.dagger:hilt-compiler:$hiltVersion")

Reply:
@Goooler I see you changed to annotationProcessor ("com.google.dagger:hilt-compiler:$hiltVersion")

But getting below exception now while launching appp
```


  Process: app.lawnchair.lawnicons, PID: 30361
                                                                                                    java.lang.RuntimeException: Unable to instantiate application app.lawnchair.lawnicons.LawniconsApplication package app.lawnchair.lawnicons: java.lang.ClassNotFoundException: Didn't find class "app.lawnchair.lawnicons.LawniconsApplication" on path: DexPathList[[zip file "/data/app/~~o-8lXcw91hImyuOEZLyQHA==/app.lawnchair.lawnicons-uObhy24i-P4ZEGnIwQesoQ==/base.apk"],nativeLibraryDirectories=[/data/app/~~o-8lXcw91hImyuOEZLyQHA==/app.lawnchair.lawnicons-uObhy24i-P4ZEGnIwQesoQ==/lib/arm64, /system/lib64, /system/system_ext/lib64]]
                                                                                                    	at android.app.LoadedApk.makeApplication(LoadedApk.java:1465)
                                                                                                    	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7540)
                                                                                                    	at android.app.ActivityThread.access$1600(ActivityThread.java:315)
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2286)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:106)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:226)
                                                                                                    	at android.os.Looper.loop(Looper.java:313)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8751)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
                                                                                                    Caused by: java.lang.ClassNotFoundException: Didn't find class "app.lawnchair.lawnicons.LawniconsApplication" on path: DexPathList[[zip file "/data/app/~~o-8lXcw91hImyuOEZLyQHA==/app.lawnchair.lawnicons-uObhy24i-P4ZEGnIwQesoQ==/base.apk"],nativeLibraryDirectories=[/data/app/~~o-8lXcw91hImyuOEZLyQHA==/app.lawnchair.lawnicons-uObhy24i-P4ZEGnIwQesoQ==/lib/arm64, /system/lib64, /system/system_ext/lib64]]
                                                                                                    	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
                                                                                                    	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
                                                                                                    	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
                                                                                                    	at android.app.AppComponentFactory.instantiateApplication(AppComponentFactory.java:76)
                                                                                                    	at androidx.core.app.CoreComponentFactory.instantiateApplication(CoreComponentFactory.java:52)
                                                                                                    	at android.app.Instrumentation.newApplication(Instrumentation.java:1177)
                                                                                                    	at android.app.LoadedApk.makeApplication(LoadedApk.java:1457)
                                                                                                    	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7540) 
                                                                                                    	at android.app.ActivityThread.access$1600(ActivityThread.java:315) 
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2286) 
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:106) 
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:226) 
                                                                                                    	at android.os.Looper.loop(Looper.java:313) 
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8751) 
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571) 
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135) 
                                                                                                    	Suppressed: java.lang.NoClassDefFoundError: Failed resolution of: Lapp/lawnchair/lawnicons/Hilt_LawniconsApplication;
                                                                                                    		at java.lang.VMClassLoader.findLoadedClass(Native Method)
                                                                                                    		at java.lang.ClassLoader.findLoadedClass(ClassLoader.java:738)
                                                                                                    		at java.lang.ClassLoader.loadClass(ClassLoader.java:363)
                                                                                                    		... 15 more
```

Reply:
Need to adopt Napt.

Reply:
Found that this is related to new Napt version 1.18 which happend on two days before. If I revert to one 1.17 it's workig fine

Reply:
Could you file an issue to Napt?

Reply:
https://github.com/sergei-lapin/napt/issues/16

Reply:
Solved in #830